client/src/components/App.jsx

```diff
@@ -42,14 +43,17 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      restaurantId: 'w3ze0',
```

Comment:
are all the restaurant ids hard coded?

Reply:
They're in the database. This is hardcoded for testing